scripts/gz10/README.md

```diff
@@ -0,0 +1,50 @@
+# GZ10 Galaxy Catalog Dataset
+
+The GZ10 catalog, provided by Leung et al. (2018), features 17,736 galaxies with labels from the Galaxy Zoo 2 project. 
+
+## Getting Started
+
+### Downloading the Dataset
+
+To download the GZ10 catalog, use the following `wget` command:
+
+```bash
+wget https://www.astro.utoronto.ca/~hleung/shared/Galaxy10/Galaxy10_DECals.h5
```

Comment:
This is the DECaLS version, while you elsewhere say it is the original GZ10 SDSS version